modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/RefNameDeclarationEnricher.java

```diff
@@ -54,8 +59,13 @@ protected ModelProperty getModelProperty(Field field) {
   }
 
   @Override
-  protected Class getAnnotation() {
-    return RefName.class;
+  protected Predicate<Field> getFieldHasAnnotationPredicate() {
+    return withAnnotation(RefName.class);
```

Comment:
shouldn't be checked the annotation org.mule.sdk.api.annotation.param.RefName here?

Reply:
This will be added when the `org.mule.sdk.api.annotation.param` package is implemented. For the scope of each task I think that it is the best to keep it as to what it is necessary for the task to be complemeted, and if a new case is taken into account, implement it extensively.

Reply:
got it, agree